src/app/pages/ParaTimesPage/TransactionAmount/FeesSection.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import { Box, Button, Collapsible, FormField, TextInput } from 'grommet'
+import { useTranslation } from 'react-i18next'
+
+type FeesSectionProps = {
+  feeAmount: string
+  feeGas: string
+  ticker: string
+}
+
+export const FeesSection = ({ feeAmount, feeGas, ticker }: FeesSectionProps) => {
+  const { t } = useTranslation()
+  const [open, setOpen] = useState(false)
+
+  return (
+    <Box pad={{ vertical: 'medium' }}>
+      <Box align="start">
+        <Button
+          margin={{ bottom: 'medium' }}
+          secondary
+          size="small"
+          onClick={() => setOpen(!open)}
```

Comment:
What happens if user opens advanced, chooses fees, and closes advanced

Reply:
It's working the same way as in the current ext. Collapse/Expand is just a visual effect with no effect on form input data. 

1) keep current 
2) disable collapsing when any input has a value and start with an open state set to true when nav between steps if there's value
3) when fee section is closed but has a value show some labels instead

Reply:
I vote disable

Another bug flow:
- withdraw from emerald: fill private key, address, amount, click advanced, fill fees
- click previous
- withdraw from cipher: advanced fees are prefilled, but not opened